atd-vze/src/views/Crashes/CrashChangeLog.js

```diff
@@ -1,243 +1,177 @@
-          <ModalBody>{this.state.modalBody}</ModalBody>
-          <ModalFooter>
-            <Button color="secondary" onClick={this.closeModal}>
-              Close
-            </Button>
-          </ModalFooter>
-        </Modal>
-      );
-      content = (
-        <>
-          <h4>Record History</h4>
-
-          <Table responsive>
-            <thead>
+          <tbody className="text-monospace">
+            {selectedChange.diffs.map(diff => (
               <tr>
-                <td>Date Edited</td>
-                <td>Updated by</td>
-                <td></td>
+                <td className="text-monospace">{diff.field}</td>
+                {!isNewRecordEvent(selectedChange) && (
+                  <td className="text-monospace">{String(diff.old)}</td>
+                )}
+                <td className="text-monospace">{String(diff.new)}</td>
               </tr>
-            </thead>
-            <tbody>
-              {this.props.data.atd_txdot_change_log.map(record => (
-                <tr key={`changelog-${record.change_log_id}`}>
-                  <td>
-                    <Badge color="warning">
-                      {formatDateTimeString(record.update_timestamp)}
-                    </Badge>
-                  </td>
-                  <td>
-                    <Badge color="danger">
-                      {String(record.updated_by || "Unavailable")}
-                    </Badge>
-                  </td>
-                  <td>
-                    <Button
-                      color="primary"
-                      size="sm"
-                      onClick={() => this.compare(record)}
-                    >
-                      Compare
-                    </Button>
-                  </td>
-                </tr>
-              ))}
-            </tbody>
-          </Table>
-        </>
-      );
-    }
-
-    // Render Box
-    return (
-      <div className="animated fadeIn">
-        {modal}
-        <Row>
-          <Col>
-            <Card>
-              <CardHeader>
-                <i className="fa fa-history" /> Change Log
-              </CardHeader>
-              <CardBody>{content}</CardBody>
-            </Card>
-          </Col>
-        </Row>
-      </div>
-    );
+            ))}
+          </tbody>
+        </Table>
+      </ModalBody>
+      <ModalFooter>
+        <Button color="secondary" onClick={() => setSelectedChange(null)}>
+          Close
+        </Button>
+      </ModalFooter>
+    </Modal>
+  );
+};
+
+/**
+ * The primary UI component which renders the change log with clickable rows
+ */
+export default function CrashChangeLog({ data }) {
+  const [selectedChange, setSelectedChange] = useState(null);
+
+  const changes = useChangeLogData(data);
+  if (changes.length === 0) {
+    return <p>No change history found</p>;
   }
-}
 
-export default CrashChangeLog;
+  return (
+    <Card>
+      <CardHeader>Record history</CardHeader>
```

Comment:
totally a thought for later on, but it is really starting to stand out to me how the card headers are overshadowed by our bolded text for field names and the column headers.